hack/generate-proto.sh

```diff
@@ -1,30 +1,63 @@
 #!/bin/bash
 set -eux -o pipefail
-go get k8s.io/code-generator/cmd/go-to-protobuf@v0.16.7-beta.0
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway@v1.12.2
-go get github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.12.2
-go get github.com/gogo/protobuf/protoc-gen-gogo@v1.3.1
-go get github.com/gogo/protobuf/protoc-gen-gogofast@v1.3.1
-go get github.com/gogo/protobuf/gogoproto@v1.3.1
-go get golang.org/x/tools/cmd/goimports
-go install k8s.io/code-generator/cmd/go-to-protobuf
-go-to-protobuf \
+
+source $(dirname $0)/library.sh
+
+header "generating proto files"
+
+if [ ! -d "${REPO_ROOT}/vendor" ]; then
```

Comment:
is this what makes it slow? could we just have these checks?

Reply:
No. The reason to make it slow is simply because of moving to `go module`, see https://github.com/kubernetes/code-generator/issues/69.

I add `go mod vendor` here just because I want to make sure `./hack/generate-proto.sh` also works, no need to run `make`.